R/get_R.R

```diff
@@ -167,15 +172,37 @@ get_R.integer <- function(x, disease = NULL, si = NULL,
 
   x_with_tail <- rep(0, last_day)
   x_with_tail[dates] <- x
-  all_lambdas <- EpiEstim::overall_infectivity(x_with_tail, si)[-1]
-  dates_lambdas <- seq_along(all_lambdas) + 1
-  x_lambdas <- all_lambdas[dates]
 
-  loglike <- function(R) {
-    if (R <= 0 ) return(-Inf)
-    sum(stats::dpois(x[-1], lambda = R * x_lambdas, log = TRUE))
+  
+  ## Important note: we cannot compute the likelihood of the first day with
```

Comment:
incidentally, I think this new version fixes https://github.com/reconhub/earlyR/issues/7
@acori to check?